### PR TITLE
MBS-12614: Better placement for Add collaborator button

### PR DESCRIPTION
**Pull request by reosarevok**
### Implement MBS-12614

The autocomplete input is very short, so it doesn't make sense for `form-row-add` to use a full standard `form-input-width` here. Since we already have `form-input-width-short` which fits better, I'm just adding a `form-row-add.short` combo to match.

image

### CODE REVIEW

**Comment by reosarevok** [comment]
@brainzbot, retest this please

File: root/static/styles/forms.less [hunk 1]
```diff
@@ -93,6 +93,8 @@ div.form-row-add {
     margin: (@form-margin / 2) 0;
 }
 
+div.form-row-add.short { width: @form-input-width-short + 2; }
```
Comment (mwiencek) [inline root/static/styles/forms.less:96]: What does the `+ 2` do? (It actually looks better if I remove that, since the buttons align perfectly.)

**Review by mwiencek — COMMENTED** [review]
I was going to ask why they aren't aligned in the screenshot you gave, but I tested it locally and the icons are aligned for me (other than the 2 pixel offset), so dunno if that was a before screenshot or what :)

File: root/static/styles/forms.less (hunk 1 above)
Comment (reosarevok) [reply root/static/styles/forms.less:96]: I just followed the +2 from form-row-add, but I'm fine with removing that if you feel it looks better :)

Comment (mwiencek) [reply root/static/styles/forms.less:96]: It looks better in FF at least.  In Chrome I get the same as your screenshot, but in FF if I remove the `+ 2` I see this, which looks perfect:
image
Dunno why the text input is 202px in FF and 158px in Chrome.

Comment (reosarevok) [reply root/static/styles/forms.less:96]: Me neither tbh. Wonder if we could port this to Autocomplete2 and get it to be consistent that way? :)

Comment (mwiencek) [reply root/static/styles/forms.less]: Pushed a commit for that.  If you'd prefer we can merge this first and I can move it into a separate PR.

**Comment by reosarevok** [comment]
For some reason this seems to be breaking removing removed collaborators from private collection subscriptions? I can't easily figure out why tbh.

**Comment by reosarevok** [comment]
Ok, the issue was with the test itself - what used to work to submit the form earlier no longer did now. Amended the test as needed, seems fine now.

**Comment by mwiencek** [comment]
Fixed the `useLayoutEffect` warning too.

**Review by mwiencek — APPROVED** [review]

**Pull request merged**